simulator/obj_canvas.py

```diff
@@ -154,7 +154,38 @@ def set_background_color(self, color):
     Args:
       color: The color to set it to. """
     self.__canvas.config(bg=color)
-
+  def draw_grid(self):
```

Comment:
From a design perspective, ```obj_canvas``` should remain as low-level as possible, preferably containing nothing but primitives. Therefore, I'd create a ```Line``` class in ```obj_canvas```, and move all the grid stuff to another class in another file. Within the grid class, you can then use ```obj_canvas.Line``` to draw lines.

It's kind of the same idea behind the ```Cube``` class using multiple invocations of ```obj_canvas.Rectangle``` in order to draw the cube.